services/core/shared/core/helpers/accountSearch.js

```diff
@@ -13,32 +13,34 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const config = require('../../../config');
-const db = require('../../database')[config.db.collections.accounts.db || config.db.defaults.db];
+// const config = require('../../../config');
```

Comment:
Please remove the file.

Reply:
Given change already done in the PR: https://github.com/LiskHQ/lisk-service/pull/338